src/asmcup/sandbox/CodeEditor.java

```diff
@@ -51,10 +51,7 @@ public void dropFiles(DropTargetDropEvent e) throws Exception {
 		@SuppressWarnings("unchecked")
 		List<File> droppedFiles = (List<File>) obj;
 
-		for (File file : droppedFiles) {
-			openFile(file);
-			return;
-		}
+		openFile(droppedFiles.get(0));

```

Comment:
This will cause an exception/error if an empty list makes it into the drag drop, which I don't know if that can happen.